lib/expo/message.ex

```diff
@@ -115,9 +115,13 @@ defmodule Expo.Message do
   """
   @spec has_flag?(t(), String.t()) :: boolean()
   def has_flag?(%mod{flags: flags} = _message, flag)
-      when mod in [Singular, Plural] and is_binary(flag) do
-    flag in List.flatten(flags)
-  end
+      when mod in [Singular, Plural] and is_binary(flag),
+      do: raw_has_flag?(flags, flag)
+
+  @doc false
```

Comment:
Why is this public and `@doc false`?

Reply:
@whatyouhide Oops, that is not needed. For `raw_append_flag` I need this to call it from the singular / plural modules.